src/NumberList/NumberList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './numberList.scss';
+
+export const NumberList = (props) => {
+  const num = [];
+  for (let i = +props.from; i <= +props.to; i++) {
+    num.push(i);
+  }
+  const items = num.map(item => <li key={item}>{item}</li>);
```

Comment:
move this string before return statement, after if, because you make unnecessary mapping (in case odd or even flags you make array mapping twice)